Add_user.css

```diff
@@ -104,44 +114,76 @@ ul li:hover ul{
 	width:100%
 }
 
+/* Set up the footer */
 .footer{
-	background-color: white;
-	grid-column-start: 1;
+	background-color: #0C59CF;
 	grid-column-end: span 4;
+	padding-top:25px;
 	text-align: center;
-	height:35px;
-	color:black;
-	padding-top:20px;
+	color:white;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+/* Re position of text in the div */
 h3{
 	margin-left:130px;
 	margin-top:30px;
 	
 }
 
+/* Re position of text in the div and change font-size, color, weight */
 p{
 	margin-left:130px;
 	margin-top:250px;
 	font-size:24px;
-	color:black;
-	font-weight: bold;
+	color:#0C59CF;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
 }
 
 #us{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
 #pa{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
I also apply the font that I decided to use for the passwords box. In this case it is open sans